test/benchmark/vitest.config.ts

```diff
@@ -1,25 +1,7 @@
 import { defineConfig } from 'vitest/config'
 
-function noop() {}
-
 export default defineConfig({
   test: {
-    update: false,
-    allowOnly: true,
-    benchmark: {
-      outputFile: './bench.json',
-      reporters: ['json', {
-        onInit: noop,
-        onPathsCollected: noop,
-        onCollected: noop,
-        onFinished: noop,
-        onTaskUpdate: noop,
-        onTestRemoved: noop,
-        onWatcherStart: noop,
-        onWatcherRerun: noop,
-        onServerRestart: noop,
-        onUserConsoleLog: noop,
-      }],
-    },
+    fileParallelism: false,
```

Comment:
Is this needed to be specified here? Without this the default value's setter would be included by tests. 

```suggestion
```

Reply:
Hmm, I don't remember why I have this here, but maybe I was testing `fileParallelism` for old fixture, which is moved out already.
I'll remove this now.